# Request
string command  # "open" or "close"
---
# Response
bool success
string message 